bigbite/image-comparison
# [17] Add resolution controls to image comparison block

## Description
[17](https://github.com/bigbite/image-comparison/issues/17) - Adds image resolution controls to the image comparison block so you can choose which size you want to use


## Change Log

- Adds a ResolutionTool component based on the one from [core](https://github.com/Wordpress/gutenberg/blob/1c3c161382e7bb157778f0d50d885324953ec076/packages/block-editor/src/components/resolution-tool/index.js) but with differences as this block didn't need some things.
- Adds @wordpress/data and @wordpress/core-data to package.json

## Steps to test

Add the image comparison block and add an image to the left and the right. 
Click on the left or right image and you will see a new setting appear on the right. 
Click a different image resolution and check the image changes in the editor
Save the post/page and view it on the front end. Inspect the photo and see the correct image size is showing http://bigbite.im/i/qfiJA0
## Screenshots/Videos

http://bigbite.im/v/EwBGS8

## Checklist:
